### PR TITLE
Video: Goodbye Monada - Hello Wing Cloud

### PR DESCRIPTION
**Pull request by dannac2**
# Docs updates should not be submitted in this repository

Our doc site content is being automatically updated from [Wing](https://github.com/winglang/wing) repository docs folder.

Please submit any changes to the docs as a PR [here](https://github.com/winglang/wing/pulls)

### CODE REVIEW

File: blog/2023-08-10-wingly-introducing-wing-cloud.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  - eyalk
+  - shaib
+tags: [wingly, winglang, video,wingcloud]
+hide_table_of_contents: true
+---
+
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/mvWv9bVyQdc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+> In this short video, Eyal and Shai give some behind the scenes details of Wing Cloud’s recent launch out of stealth, with a $20 million seed round! Listen in as they discuss the Vision of Wing Cloud, and how it relates to our ongoing commitment to Winglang.
```
Comment (dannac2) [inline blog/2023-08-10-wingly-introducing-wing-cloud.md:13]:
```suggestion
> In this short video, Eyal and Shai give some behind the scenes details of Wing Cloud’s recent launch out of stealth, with a $20 million seed round! Listen in as they discuss the vision of Wing Cloud, and how it relates to our ongoing commitment to Winglang as an open-source project.
```

File: blog/2023-08-10-wingly-introducing-wing-cloud.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  - eyalk
+  - shaib
+tags: [wingly, winglang, video,wingcloud]
+hide_table_of_contents: true
+---
+
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/mvWv9bVyQdc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+> In this short video, Eyal and Shai give some behind the scenes details of Wing Cloud’s recent launch out of stealth, with a $20 million seed round! Listen in as they discuss the vision of Wing Cloud, and how it relates to our ongoing commitment to Winglang as an open-source project.
```
Comment (eladb) [inline blog/2023-08-10-wingly-introducing-wing-cloud.md:13]:
```suggestion
> In this short video, Eyal and Shai give some behind the scenes details of Wing Cloud’s recent launch out of stealth, with a $20 million seed round. Listen in as they discuss the vision of [Wing Cloud](https://wing.cloud) as well as our ongoing commitment to Winglang as an open-source project.
```

**Review by eladb — COMMENTED** [review]
@dannac2 please also update the pull request title

File: blog/2023-08-10-wingly-introducing-wing-cloud.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+title: "Video: Goodbye Monada - Hello Wing Cloud"
```
[the remark this answers is not in the document]
Comment (dannac2) [reply blog/2023-08-10-wingly-introducing-wing-cloud.md:2]: yes

**Review by eladb — APPROVED** [review]

**Pull request merged**